operators/pkg/tenant-controller/tenant_controller.go

```diff
@@ -517,6 +532,32 @@ func (r *TenantReconciler) createOrUpdateTnPersonalNFSVolume(ctx context.Context
 			return err
 		}
 		klog.Infof("PVC Secret for tenant %s %s", tn.Name, pvcSecOpRes)
+
+		val, ok := pvc.Labels[ProvisionJobLabel]
+		if !ok || val != ProvisionJobValueOk {
+			chownJob := batchv1.Job{ObjectMeta: metav1.ObjectMeta{Name: pvc.Name + "-provision", Namespace: pvc.Namespace}}
+
+			chownJobOpRes, err := ctrl.CreateOrUpdate(ctx, r.Client, &chownJob, func() error {
+				r.updateTnProvisioningJob(&chownJob, &pvc)
+				return ctrl.SetControllerReference(tn, &chownJob, r.Scheme)
+			})
+			if err != nil {
+				klog.Errorf("Unable to create or update PVC Provisioning Job for tenant %s -> %s", tn.Name, err)
+				return err
+			}
+			klog.Infof("PVC Provisioning Job for tenant %s %s", tn.Name, chownJobOpRes)
+
+			if chownJob.Status.Succeeded == 1 {
+				pvc.Labels[ProvisionJobLabel] = ProvisionJobValueOk
+				if err := r.Update(ctx, &pvc); err != nil {
+					klog.Errorf("PVC Provisioning Job failed to update PVC labels for tenant %s", tn.Name)
+				}
+
+				klog.Infof("PVC Provisioning Job completed for tenant %s", tn.Name)
+			} else if chownJob.Status.Failed == 1 {
+				klog.Errorf("PVC Provisioning Job failed for tenant %s", tn.Name)
```

Comment:
probably better to move into a warning (if exists)
```suggestion
				klog.Warningf("PVC Provisioning Job failed for tenant %s", tn.Name)
```